templates/shift-scheduling/schema.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/nextmv-io/sdk/mip"
+)
+
+// output holds the output data of the solution.
+type output struct {
+	AssignedShifts        []outputAssignment `json:"assigned_shifts"`
+	NumberAssignedWorkers int                `json:"number_assigned_workers"`
+}
+
+// options holds custom configuration data.
+type options struct {
```

Comment:
Would be great to think about a possible nested structure here to have nicer looking CMD args.
E.g.
```go
penalty {
  oversupply
  undersupply
}
```
results in `-penalty.oversupply=1000` which looks nicer than `-oversupplypenalty=1000` IMO

Reply:
@nilsbeck do you think nesting the other options makes sense as well? I don't have a good suggestion though 🤔

```
Limits {
   Shift {
       MinHours int
       MaxHours int
   }
   Week {
      MaxHours int
   }
   Day {
      MaxHours int
   }
}
```

In addition: why is the type `time.Duration` and not `int`?